ibm_catalog.json

```diff
@@ -111,11 +111,79 @@
               },
+              {
+                "key": "secrets_manager_region",
+                "required": true,
+                "virtual": true,
+                "default_value": "us-south",
+                "description": "The region to provision a new Secrets Manager instance in.",
+                "options": [
+                  {
+                    "displayname": "Osaka (jp-osa)",
+                    "value": "jp-osa"
+                  },
+                  {
+                    "displayname": "Sydney (au-syd)",
+                    "value": "au-syd"
+                  },
+                  {
+                    "displayname": "Tokyo (jp-tok)",
+                    "value": "jp-tok"
+                  },
+                  {
+                    "displayname": "Frankfurt (eu-de)",
+                    "value": "eu-de"
+                  },
+                  {
+                    "displayname": "London (eu-gb)",
+                    "value": "eu-gb"
+                  },
+                  {
+                    "displayname": "Madrid (eu-es)",
+                    "value": "eu-es"
+                  },
+                  {
+                    "displayname": "Dallas (us-south)",
+                    "value": "us-south"
+                  },
+                  {
+                    "displayname": "Toronto (ca-tor)",
+                    "value": "ca-tor"
+                  },
+                  {
+                    "displayname": "Washington DC (us-east)",
+                    "value": "us-east"
+                  },
+                  {
+                    "displayname": "Sao Paulo (br-sao)",
+                    "value": "br-sao"
+                  }
+                ]
+              },
+              {
+                "key": "secrets_manager_service_plan",
+                "required": true,
+                "virtual": true,
+                "type": "string",
+                "options": [
+                  {
+                    "displayname": "Standard",
+                    "value": "standard"
+                  },
+                  {
+                    "displayname": "Trial",
+                    "value": "trial"
+                  }
+                ],
+                "default_value": "basic",
+                "description": "The pricing plan to use when provisioning a Secrets Manager instance. Possible values: `standard`, `trial`. You can create only one Trial instance of Secrets Manager per account. Before you can create a new Trial instance, you must delete the existing Trial instance and its reclamation. [Learn more](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-create-instance&interface=ui#upgrade-instance-standard)."
+              },
               {
                 "key": "ibmcloud_cis_api_key"
               },
               {
-                "key": "internet_services_crn"
+                "key": "internet_services_crn",
+                "required": true
```

Comment:
Is this required or not? In terraform it has a default value of null, so its not required. But here you are adding it as required. The variable description is missing the information on when / why a user would need or want to enter a value for this.